iina/hi.lproj/PrefNetworkViewController.strings

```diff
@@ -0,0 +1,78 @@
+
+/* Class = "NSTextFieldCell"; title = "Default: 153600"; ObjectID = "0bT-f6-rXb"; */
+"0bT-f6-rXb.title" = "डिफ़ॉल्ट: 153600";
+
+/* Class = "NSTextFieldCell"; title = "Default cache size (KB):"; ObjectID = "0yu-5Q-363"; */
+"0yu-5Q-363.title" = "डिफ़ॉल्ट कैश आकार (KB):";
+
+/* Class = "NSTextFieldCell"; title = "youtube-dl:"; ObjectID = "1Ex-0X-pjo"; */
+"1Ex-0X-pjo.title" = "youtube-dl:";
+
+/* Class = "NSTextFieldCell"; title = "Custom youtube-dl path:"; ObjectID = "2qf-cC-Lfi"; */
+"2qf-cC-Lfi.title" = "कस्टम youtube-dl पथ:";
+
+/* Class = "NSTextFieldCell"; title = "Default: 153600"; ObjectID = "3os-7u-lv8"; */
+"3os-7u-lv8.title" = "डिफ़ॉल्ट: 153600";
+
+/* Class = "NSMenuItem"; title = "TCP"; ObjectID = "6JF-48-aCE"; */
+"6JF-48-aCE.title" = "TCP";
+
+/* Class = "NSTextFieldCell"; title = "HTTP proxy:"; ObjectID = "9ut-ID-ffZ"; */
+"9ut-ID-ffZ.title" = "HTTP proxy:";
```

Comment:
Ditto

Reply:
@alejx Hi TCP & HTTP would stay in english right as their is no hindi equivalent & would it be considered noun.